schemas.deprecated/src/registry_schemas/example_data/schema_data.py

```diff
@@ -323,7 +323,8 @@
 
 INCORPORATION = {
         'nameRequest': {
-            'legalType': 'BC'
+            'legalType': 'BC',
+            'nrNumber': 'NR1234567'
         },
```

Comment:
The schemas have recently been moved to their own repo business-schemas, so this change will need to move there. It can be its own PR (since different repo). 
https://github.com/bcgov/business-schemas

If you're working on this in your local branch again, you'll also want to change your requirements file to the new repo, but it's already changed in master so you can just rebase and re-install and you should be fine.

Reply:
ohh right, i will update this and submit a new PR to that repo. Totally forget this all moved recently. Thanks for the heads up!